File: backend/notification/src/main/kotlin/com/linkedout/notification/model/Notification.kt
package com.linkedout.notification.model

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table
import java.time.LocalDateTime
import java.util.*

@Table("notification")
data class Notification(
    @Id
    val id: UUID,
    @Column("seasonworkerid")
    val seasonworkerId: UUID,
    val title: String,
    val content: String,
    val created: LocalDateTime
)

File: front/src/models/entities/notification.ts
/**
 * Notification.
 */
export type Notification = {
  /**
   * The id of the notification.
   */
  id: string;

  /**
   * The title of the notification.
   */
  title: string;

  /**
   * The content of the notification.
   */
  content: string;

  /**
   * The date of the notification.
   * Formatted as an ISO 8601 string.
   */
  sentAt: string;
};
